.github/workflows/lint.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         name: Checkout Code
 
       - name: Set up Ruby
-        uses: ruby/setup-ruby@v1.154.0
```

Comment:
any reason why we're removing those here and below?

Reply:
Just following recommendation to get latest ruby version always.. https://github.com/ruby/setup-ruby?tab=readme-ov-file#versioning